draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -173,13 +177,20 @@ Security Considerations
 
 ## Additions to Existing Registries
 
+### COSE Header Parameters registry
+
+This document requests IANA to add the following new value to the 'COSE Header Parameters' registry:
+
+* Label: TBD_2
```

Comment:
I assume this is requesting a tag for the `proof type` for CCF?

See https://github.com/ietf-scitt/draft-steele-cose-merkle-tree-proofs/pull/18#issuecomment-1620457971

The current Cometre draft takes the 2 layer approach, you would encode a CCF inclusion proof using the TBD_2 tag of cometre... it would be understood as CCF proof, based on TBD_1 tag of cometer using your TBD_1 tag in this draft (you can request assignment of 2 for CCF).

Reply:
we can iron out registration nits during the submission moratorium